WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -256,6 +256,42 @@ public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int? fe
 		return smartTx.Transaction.ToHex();
 	}
 
+	[JsonRpcMethod("buildtransactionwithoutoverpaymentprotection")]
```

Comment:
What a beauty...

Reply:
maybe remove _transaction_?
-> `buildwithoutoverpaymentprotection`
as it's `build` method and not `buildtransaction`, so this makes more sense I think

Reply:
buildunsafetransaction?